### ISSUE TITLE
Loader for joint training

### ISSUE THREAD

**Issue by yonghole** [cvlab-stonybrook/DewarpNet#27]
We use separate loader for each network.
I combined the code, but do i need to fix the loader to bring materials for each network?

**Comment by sagniklp** [comment]
yeah, input is the image, load the wc ground-truth, and bm ground-truth.

**Comment by YongSangHo** [comment]
What about the alb?

**Comment by YongSangHo** [comment]
What about the alb? I thought alb is also used for recon_loss in texture mapping network.
Also, just make total loss 0.5 * wc_loss + 0.5 * bm_loss and back propagate. Right?

**Comment by Wanwan1996** [comment]
Can you provide the code for joint training, thank you very much ！

**Comment by sagniklp** [comment]
Sorry for swamping this, will try to do this by the weekend. 
You can also do it pretty easily, just combine two training codes and rename the variables accordingly.
In case you do it, you're welcome to open a pull request, I'll verify & merge.

**Comment by Wanwan1996** [comment]
I have merged, ok, today I will submit it

**Comment by ARnnn** [comment]
Can the code of joint training be released？

**Comment by irisXcoding** [comment]
> What about the alb? I thought alb is also used for recon_loss in texture mapping network. Also, just make total loss 0.5 * wc_loss + 0.5 * bm_loss and back propagate. Right?

hi, could you please tell me if this way works: just make total loss 0.5 * wc_loss + 0.5 * bm_loss and back propagate
so many thx~
